### DISCUSSION TITLE
Mainnet Stable Release 1.3.1.1

### DISCUSSION THREAD

**Discussion by lk86** [MinaProtocol/mina#11611] [Announcements]
The O(1) Labs team is proud to announce that 1.3.1.1 is ready for use in production on Mainnet! This release includes a crucial bug discovered during the release process for 1.3.2 which we decided to fix and release immediately in an abundance of caution. This release is identical to 1.3.1 with the exception of addressing this bug, and it is recommended that all users upgrade at their earliest convenience.

If you run into any issues, please include `Release: 1.3.1.1` in the name of any issues you create. Please report any issues you encounter [here](https://github.com/MinaProtocol/mina/issues/new/choose) or join the Discussion below!

### Bug Fixes
- Resolve pickles inconsistencies #11592

### Complete Changelog since 1.3.1:
https://github.com/MinaProtocol/mina/compare/1.3.1...1.3.1.1

### Debian Packages:
**Debian Repository:**
First, set up and update the `stable` Debian Repository for your platform with the following commands:
```
sudo rm /etc/apt/sources.list.d/mina*.list
echo "deb [trusted=yes] http://packages.o1test.net $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/mina.list
sudo apt-get update
```
Then, install the package(s) that you need:
**Mina Daemon:** `sudo apt-get install -y mina-mainnet=1.3.1.1-f361ba1`
**Mina Archive:** `sudo apt-get install -y mina-archive=1.3.1.1-f361ba1`
**New Mina Archive Schema:** [create_schema.sql](https://github.com/MinaProtocol/mina/blob/f361ba19d78ba930b5b551f58d1a82942c0f724b/src/app/archive/create_schema.sql)

### Docker Images:
Every docker image is built for each of the supported platforms, replace `CODENAME` in the images below with your choice of `bionic`, `focal`, `stretch`, `buster`, or `bullseye` to select the base docker image.

**Mina Daemon:** `minaprotocol/mina-daemon:1.3.1.1-f361ba1-CODENAME-mainnet`
**Mina Archive:** `minaprotocol/mina-archive:1.3.1.1-f361ba1-CODENAME`
**New Mina Archive Schema:** [create_schema.sql](https://github.com/MinaProtocol/mina/blob/f361ba19d78ba930b5b551f58d1a82942c0f724b/src/app/archive/create_schema.sql)
**Mina Rosetta:** `minaprotocol/mina-rosetta:1.3.1.1-f361ba1-CODENAME`

### Sandbox Node: For testing in an isolated, single-node network without snarks
As of 1.2.0, the demo functionality is built into the standard daemon container, via the environment variable `RUN_DEMO=true`.
Just `docker run -it --name mina-demo -e RUN_DEMO=true minaprotocol/mina-daemon:1.3.1.1-f361ba1-CODENAME-devnet` and go!

### Step by Step Guide:
Check out [our documentation](https://docs.minaprotocol.com/en/using-mina/connecting) for complete instructions on using this version to connect to Mainnet.
*Docker only* Make sure to run with `--peer-list-url https://storage.googleapis.com/mina-seed-lists/mainnet_seeds.txt` or add `PEER_LIST_URL=https://storage.googleapis.com/mina-seed-lists/mainnet_seeds.txt` to your `.mina-env`.

If you are running the correct version on the correct network **MAINNET**, `mina client status` will show:
```
Chain id:     5f704cc0c82e0ed70e873f0893d7e06f148524e3f0bdae2afb02e7819a0c24d1
Git SHA-1:    f361ba19d78ba930b5b551f58d1a82942c0f724b
```

<hr /><em>This discussion was created from the release <a href='https://github.com/MinaProtocol/mina/releases/tag/1.3.1.1'>Mainnet Stable Release 1.3.1.1</a>.</em>

**Comment by zongjiancai** [comment]
是否合并
